### ISSUE TITLE
Importing Contacts into Contacts+ 1.0.4

### ISSUE THREAD

**Issue by BenBenna** [libasys/contactsplus#47]
Hello,
I want to import contacts into my owncloud as described in "Using the Contacts App" in the "owncloud User Manual". This is as it is described there:
![importowncloudcontacts](https://cloud.githubusercontent.com/assets/14091725/9630582/958f3de4-517b-11e5-9e8b-40003dea6542.PNG)
This is as it looks on my owncloud 8.1.1.?
![importowncloud](https://cloud.githubusercontent.com/assets/14091725/9630761/b2378e64-517c-11e5-971a-04d2e66426f4.PNG)
In other words I do not really see a possibility to import my Thunderbird contacts. Only create new ones.
Is there any help? Is there a way to import contacts into Contacts+ 1.0.4?
How?

**Comment by deMattin** [comment]
Contacts+ isn't the same as contacts!

It's a really simple way to do it in contacts+:
1. Open contacts+ app
2. Drag and drop your vcf-file right, where you see your contacts.
3. A message box appears, asking in which addressbox you want to import ...
(4. You also have the possibility to clear all contacts in this addressbook before importing)

**Comment by BenBenna** [comment]
Thanks very much for the clarifying comment!
Obviously this is the way to do it with contacts+.
Unfortunately I don't find a way to convert my Thunderbird addresses into vcf-format.
Are there any alternativ ways to populate my contacts+ - aside manually entering the addresses?
Thanks

**Comment by hitam4450** [comment]
Extensions for Thunderbird available :

![oc812g](https://cloud.githubusercontent.com/assets/9738727/9640842/d2a458da-51b3-11e5-9f55-4a163d446e3d.png)

How about this one ??

regards, hitam4450

**Comment by BenBenna** [comment]
Thank you, hitam4450
but I do not find any
regards
BenBenna

**Comment by hitam4450** [comment]
See my comment! above!

**Comment by BenBenna** [comment]
Well yes - but as far as I can see there is no such extension or app.
Thanks for your help
BenBenna

**Comment by hitam4450** [comment]
Check out the extensions for Thunderbird on the net!

**Comment by BenBenna** [comment]
using google as an intermediary worked and populated my contacts+ as described by deMattin.
Danke Martin
BenBenna
